modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/AbstractMultiNodeBenchmark.java

```diff
@@ -90,25 +90,35 @@ public void nodeSetUp() throws Exception {
         startCluster();
 
         try {
-            var queryEngine = igniteImpl.queryEngine();
+            // Create a default zone on the cluster's start-up.
+            createDefaultZoneOnStartup();
 
-            var createZoneStatement = "CREATE ZONE IF NOT EXISTS " + ZONE_NAME + " WITH partitions=" + partitionCount()
-                    + ", replicas=" + replicaCount() + ", storage_profiles ='" + DEFAULT_STORAGE_PROFILE + "'";
-
-            getAllFromCursor(
-                    await(queryEngine.queryAsync(
-                            SqlPropertiesHelper.emptyProperties(), igniteImpl.observableTimeTracker(), null, null, createZoneStatement
-                    ))
-            );
-
-            createTable(TABLE_NAME);
+            // Create a default table on the cluster's start-up.
+            createDefaultTableOnStartup();
         } catch (Throwable th) {
             nodeTearDown();
 
             throw th;
         }
     }
 
+    protected void createDefaultZoneOnStartup() {
+        var queryEngine = igniteImpl.queryEngine();
+
+        var createZoneStatement = "CREATE ZONE IF NOT EXISTS " + ZONE_NAME + " WITH partitions=" + partitionCount()
+                + ", replicas=" + replicaCount() + ", storage_profiles ='" + DEFAULT_STORAGE_PROFILE + "'";
+
+        getAllFromCursor(
```

Comment:
I wonder why we don't create a zone via public SQL API. Does it make sense to ask the guys who wrote this initially, and if there is no good reason for this, to switch to public API usage? It would make it more difficult to break something later accidentally

Reply:
Switched to using public API.